### ISSUE TITLE
No bundle URL present in build 3752

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#1436]
2.4.0 (3752) crashes from launch.

> RCTFatalException: No bundle URL present…jsbundle file in your application bundle in mh_execute_header

**Comment by hawkrives** [comment]
I guess step one is to do a fresh build of master locally. [3752](https://travis-ci.org/StoDevX/AAO-React-Native/builds/265408087) was built from https://github.com/StoDevX/AAO-React-Native/commit/be61e191b0c2dccea1747d295c0088704d9de5db.

**Comment by drewvolz** [comment]
I've now successfully built  [3752](https://travis-ci.org/StoDevX/AAO-React-Native/builds/265408087) from https://github.com/StoDevX/AAO-React-Native/commit/be61e191b0c2dccea1747d295c0088704d9de5db for iOS 10.3 on both Release and Development locally.

**Comment by hawkrives** [comment]
My next step would be to check the Travis logs for 3752.3 and see if there's anything weird in it. 

Because there were no substantive changes from https://github.com/StoDevX/AAO-React-Native/compare/9679c0d8ee4cfa7030e83d9e47dcbab1ff615e67...be61e191b0c2dccea1747d295c0088704d9de5db.

**Comment by hawkrives** [comment]
Here's a clue: 

This is a [build log for CARLS](https://travis-ci.org/carls-app/carls/jobs/266580604#L3275-L3282) (I truly believe this is the exact same problem that AAO is having).

Take a look at the timestamps:

```
[00:15:41]: ▸ Generating 'CARLS.app.dSYM'
[00:15:41]: ▸ Running script 'Bundle Data Files'
[00:15:45]: ▸ Running script 'Bundle React Native code and images'
[00:16:00]: ▸ the transform cache was reset.
[00:16:02]: ▸ Touching CARLS.app
[00:16:02]: ▸ Signing /Users/travis/Library/Developer/Xcode/DerivedData/CARLS-grsdkuczesfovtfkfgcwidnmnsid/Build/Intermediates/ArchiveIntermediates/CARLS/InstallationBuildProductsLocation/Applications/CARLS.app
[00:16:03]: ▸ Touching CARLS.app.dSYM
[00:16:04]: ▸ Archive Succeeded
```

If we look at an excerpt of a [working build from AAO](https://travis-ci.org/StoDevX/AAO-React-Native/jobs/262071592#L3759-L3766), we see a much longer time period between "transform cache reset" and "touching app":

```
[02:24:40]: ▸ Generating 'AllAboutOlaf.app.dSYM'
[02:24:41]: ▸ Running script 'Bundle Data Files'
[02:24:43]: ▸ Running script 'Bundle React Native code and images'
[02:25:08]: ▸ the transform cache was reset.
[02:28:07]: ▸ Touching AllAboutOlaf.app
[02:28:08]: ▸ Signing /Users/travis/Library/Developer/Xcode/DerivedData/AllAboutOlaf-axgziitukyyjlyfvuepevdfannhu/Build/Intermediates/ArchiveIntermediates/AllAboutOlaf/InstallationBuildProductsLocation/Applications/AllAboutOlaf.app
[02:28:09]: ▸ Touching AllAboutOlaf.app.dSYM
[02:28:10]: ▸ Archive Succeeded
```

2 seconds vs. 3 minutes? That sounds like the bundle isn't getting built anymore, for some reason.

**Comment by hawkrives** [comment]
On my new [test CARLS build](https://travis-ci.org/carls-app/carls/jobs/266654627#L3279-L3286), I'm back to 3 minutes for the bundling:

```
[02:19:56]: ▸ Generating 'CARLS.app.dSYM'
[02:19:56]: ▸ Running script 'Bundle Data Files'
[02:19:59]: ▸ Running script 'Bundle React Native code and images'
[02:20:11]: ▸ the transform cache was reset.
[02:23:27]: ▸ Touching CARLS.app
[02:23:27]: ▸ Signing /Users/travis/Library/Developer/Xcode/DerivedData/CARLS-grsdkuczesfovtfkfgcwidnmnsid/Build/Intermediates/ArchiveIntermediates/CARLS/InstallationBuildProductsLocation/Applications/CARLS.app
[02:23:28]: ▸ Touching CARLS.app.dSYM
[02:23:29]: ▸ Archive Succeeded
```

**Comment by hawkrives** [comment]
This has been fixed by #1449. If it comes back, we'll reopen.
